english/lawn-mower/lawn-mower.md

```diff
@@ -6,6 +6,6 @@ You pull and pull and pull on the cord, but it just won't start.
 
 What do you do?
 
-Buy a new one!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.